lib/multiple_search_selection.dart

```diff
@@ -579,32 +586,36 @@ class _MultipleSearchSelectionState<T> extends State<MultipleSearchSelection<T>>
     pickedItems.addAll(widget.initialPickedItems ?? []);
   }
 
+  List<T> _searchAllItems(String query, { bool fuzzy = false }) {
```

Comment:
the use of a fuzzy search or not is configurable only because there were some places that a fuzzy search wasn't happening (even if `fuzzySearch != FuzzySearch.none`), but maybe a fuzzy search should always happen?